sentry-ruby/lib/sentry/utils/real_ip.rb

```diff
@@ -37,7 +39,7 @@ def calculate_ip
         # Could be a CSV list and/or repeated headers that were concatenated.
         client_ips    = ips_from(@client_ip)
         real_ips      = ips_from(@real_ip)
-        forwarded_ips = ips_from(@forwarded_for)
+        forwarded_ips = ips_from(@forwarded_for).reverse
```

Comment:
can you add a comment on why we need to call `reverse` here?

Reply:
👍  Comment added. 

From the Mozilla [docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For):
> If a request goes through multiple proxies, the IP addresses of each successive proxy is listed. This means, the right-most IP address is the IP address of the most recent proxy and the left-most IP address is the IP address of the originating client.

When processing these values for a client IP address, one typically starts from the rightmost value (the last proxy) and walks backward until the first IP not in a trusted IP range is found. This is used as the client IP address since nothing before that can be "trusted" to have recorded legitimate values.

The ActiveDispatch [code](https://github.com/rails/rails/blob/fe76a95b0d252a2d7c25e69498b720c96b243ea2/actionpack/lib/action_dispatch/middleware/remote_ip.rb#L118) reverses the list prior to processing, for example.